msteams-platform/toolkit/Teams-Toolkit-CLI.md

```diff
@@ -1,23 +1,516 @@
+
+## Interactive mode
+
+`teamsapp` command runs in interactive mode by default. You can work in non-interactive mode by setting `--interactive` flag to `false`.
+
+## `teamsapp doctor`
+
+Prerequiste checker for building Microsoft Teams apps.
+
+## `teamsapp new`
+
+By default, `teamsapp new` is in interactive mode and guides to create new Teams application. You can work in non-interactive mode by setting `--interactive` flag to `false`.
+
+| Sub command | Description |
+|:----------------  |:-------------|
+| `teamsapp new sample` | Create an app from an existing sample. |
+| `teamsapp list samples` | List all the available samples. |
+
+### Parameters for `teamsapp new`
+
+| Parameter | Required | Description |
+|:---------------- |:-------------|:-------------|
+|`--app-name -n` | Yes| Name of your Teams application.|
+|`--capability -c`| Yes| Choose Teams application capabilities, the available options are `bot`, `ai-bot`, and `ai-assistant-bot`. Use `teamsapp list templates` to see all the available options.|
+|`--programming-language -l`| No| Programming language for the project. The available options are `javascript`, `typescript`, and `csharp`. The default value is `javascript`.|
+|`--folder -f`| No | Project directory. A sub folder with your app name is created under this directory. The default value is `./`.|
+|`--spfx-framework-type -k`| No| Applicable if `framework` capability is selected. The available options are `minimal`, `react` and `none`, and the default value is `react`.|
+|`--spfx-web part-name -w`| No | Name for SharePoint Framework Web Part. The default value is helloworld.|
+|`--spfx-folder`| No | Directory or Path that contains the existing SharePoint Framework solution.|
+|`--me-architecture -m`| No | Architecture of search based message extension. The available options are `new-api`, `api-spec`, and `bot`. The default value is `new-api`. |
+|`--openapi-spec-location -a`| No | OpenAPI description document location. |
+|`--api-operation -o`| No | Select Operation(s) to interact with Teams. |
+|`--bot-host-type-trigger -t`| No | Applicable if `Chat Notification Message` capability is selected. The options are `http-restify`, `http-webapi`, and `http-and-timer-functions`. The default value is `http-restify`.|
+|`--spfx-solution -s`| No| Create a new or import an existing SharePoint Framework solution. The available options are `new` and `import`, and the default value is `new`.|
+|`--spfx-install-latest-package`| No| Install the latest version of SharePoint Framework. The default value is `true`.|
+
+:::image type="content" source="../assets/images/teams-toolkit-cli/teamsapp-new-parameters.png" alt-text="Screenshot shows the teamsapp new parameters.":::
+
+### Scenarios for `teamsapp new`
+
+You can use interactive mode to create a Teams app. The following list provides scenarios on controlling all the parameters with `teamsapp new`:
+
+* HTTP triggered notification bot with timer triggered interactive mode.
```

Comment:
For interactive mode, `teamsapp new` is enough.

Reply:
Updated.